src/kOS.Safe/Exceptions/KOSSerializationException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
What is the extra benefit of having this derived class if we arent doing anything with it?